src/SupervisorOptions.php

```diff
@@ -48,6 +48,13 @@ class SupervisorOptions extends WorkerOptions
      */
     public $minProcesses = 1;
 
+    /**
+     * The process niceness.
+     *
+     * @var int
+     */
+    public $nice = 0;
```

Comment:
This is problematic:
- a niceness of `0` is a valid value
- the (linux) default for niceness is usually `10`

IMHO the correct way is to use `null` here.